src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.html

```diff
@@ -24,7 +24,9 @@
             </mat-form-field>
           </div>
           <div class="error-text" *ngIf="maxArea?.invalid && maxArea?.touched">
-            Max acres needs to be at least 500 acres
+            Max acres needs to be between
```

Comment:
I like the simplicity of this. I feel like we should explain why we're enforcing min and max values, but I don't know what wording to use. Maybe we can raise that question to other people, but it's great that we'd now be able to show this in the UI while we do.

Reply:
I asked for some help on wording on the story and we landed on this, but you bring a good point that its not very clear on why this is enforced. Maybe that should be explained somewhere else and not on the error itself? Could be a good improvement when we redesign this form